### ISSUE TITLE
dynamicly change map height!?

### ISSUE THREAD

**Issue by dnsBlah** [neveldo/jQuery-Mapael#216]
Hi All,

can you please help me out here.
I'm using mapael in one of my projects, but it's not showing up quite right.
I need it to use the whole height of it's parent's container field.

But it just uses what mapael 'needs'

So when I'm zooming in, the map is just cut-off.

Please see this screenshot:
http://bayimg.com/bAIEJaagH

I made the maparea green, and you see straight away what I mean, it's not adjusting.
Of course, the wider my screen gets, the higher the maparea gets, but still leaves me gaps and its not quite ... responsive

please help!

If possible I'd even like to center the map vertically over it's area.

**Comment by neveldo** [comment]
Hello,

By default, the map width fits automatically to the width of its container, and the height will be set in a way to preserve the aspect ratio the map.

If you need the map to fit with the height of its container instead of the width, you will need to play with the "resize" event. Here is a thread that contains many usefull information : https://github.com/neveldo/jQuery-Mapael/issues/197. Here is a JSFiddle where the map height fit with the windows height and preserving the aspect ratio : http://jsfiddle.net/neveldo/mrga03aj/ . Without preserving the aspect ratio : http://jsfiddle.net/neveldo/6f6bh88c/3/ .

**Comment by dnsBlah** [comment]
I've been playing around with this indeed. However it seemed that it only works while resizing, and when I let go of the mouse it just jumped back again.

I removed my initial zoom trigger (te make sure that plots are scaled) and it did not work on load. (ofcourse, so I triggered this `paper.setSize(winW, winH);
                            paper.canvas.setAttribute('preserveAspectRatio', 'none');` inside the init itself, and it just worked out fine. (however out of scale) so after a init zoom again, it's perfect!

Many many thanks Neveldo!
To bad I can only give 1 thumb :D
